tests/zenko_tests/node_tests/utils/getWebIdentityToken.js

```diff
@@ -0,0 +1,85 @@
+const querystring = require("querystring");
+const http = require("http");
+const assert = require('assert');
+
+const USER_1_PASSWORD = process.env.KEYCLOAK_TEST_PASSWORD || '123';
+const HOST_1_URL = process.env.KEYCLOAK_TEST_HOST || 'http://keycloak.zenko.local';
+const HOST_1_PORT = parseInt(process.env.KEYCLOAK_TEST_PORT, 10) || 80;
+const REALM_NAME = process.env.KEYCLOAK_TEST_REALM_NAME || 'zenko';
+const KEYCLOAK_PATH = `/auth/realms/${REALM_NAME}/protocol/openid-connect/token`;
+const CLIENT_ID = process.env.KEYCLOAK_TEST_CLIENT_ID || 'zenko-ui';
+const GRANT_TYPE = process.env.KEYCLOAK_TEST_GRANT_TYPE || 'password';
+
+
+function getTokenForIdentity(identity, callback) {
+    getWebIdentityToken(identity, USER_1_PASSWORD, HOST_1_URL,
+        HOST_1_PORT, KEYCLOAK_PATH, CLIENT_ID, GRANT_TYPE, (err, token) => {
+            assert(err === null);
+            callback(err, token);
+        });
+}
+
+
+/**
+ * HTTP client to request JWT token given the username and password.
+ *
+ * @param {string} username - username of user requesting token
+ * @param {string} password - password of user requesting token
+ * @param {string} host - host URL of keycloak service
+ * @param {number} port - port of keycloak service
+ * @param {string} path - path of keycloak service authentication API
+ * @param {string} client_id - id of the client of the user
+ * @param {string} grant_type - grant of the user
+ * @param {function} callback - callback function called with error or result
+ * @returns {undefined} undefined
+ */
+function getWebIdentityToken(username, password, host, port, path,
```

Comment:
I think there is a lot of code duplicate here, you can probably use
`makeGetRequest` and `getResponseBody` from [request.js](https://github.com/scality/Zenko/blob/development/2.3/tests/zenko_tests/node_tests/utils/request.js)

Reply:
this `getWebIdentityToken` function is I copied from Vault project, I think it's fine.
`makeGetRequest` from request.js is mainly for cloudserver calls, here this call is for keycloak endpoint, if we really want to reuse this func, we may break a lot of current codes